Source/OxyPlot/Axes/Axis.cs

```diff
@@ -718,25 +724,24 @@ public virtual void CoerceActualMaxMin()
                 this.ActualMaximum = this.ActualMinimum + 100;
             }
 
-            // Coerce the minimum range
             var range = this.ActualMaximum - this.ActualMinimum;
+
+            // Coerce the minimum range
             if (range < this.MinimumRange)
+            { 
+                var average = (this.ActualMaximum + this.ActualMinimum) * 0.5;
+                var delta = this.MinimumRange * 0.5;
```

Comment:
This explains what the code was intended for. However, i think the fix for #550 broke the intended feature of the MinimumRange property. In the example "MinimumRange with Minimum" the MinimumRange is set to 5 but the user can clearly zoom beyond 5 units on the axis. I am not sure what to do about this.